insights/core/__init__.py

```diff
@@ -426,6 +427,120 @@ def _scan(self):
 
         cls.scan(result_key, _scan)
 
+    def _get_after(self, timestamp, time_format='%Y-%m-%d %H:%M:%S', lines=None):
```

Comment:
Not clear why this needs to be a private method? If the defaults are approriate for the logfile then the base method should work right?  Also, because this is private it is not included in the documentation so a parser developer would have to look at the code to know that it is available.